src/commands/api-mesh/log-list.js

```diff
@@ -0,0 +1,110 @@
+const { Command } = require('@oclif/core');
+
+const logger = require('../../classes/logger');
+const { initSdk, initRequestId } = require('../../helpers');
+const { ignoreCacheFlag, jsonFlag, fileNameFlag } = require('../../utils');
+const { getMeshId, listLogs } = require('../../lib/devConsole');
+const { appendFileSync, existsSync } = require('fs');
+const { ux } = require('@oclif/core/lib/cli-ux');
+const path = require('path');
+
+require('dotenv').config();
+class ListLogsCommand extends Command {
+	//static args = [{ name: 'file' }];
+	static flags = {
+		ignoreCache: ignoreCacheFlag,
+		json: jsonFlag,
+		filename: fileNameFlag,
+	};
+
+	static enableJsonFlag = true;
+
+	async run() {
+		await initRequestId();
+
+		logger.info(`RequestId: ${global.requestId}`);
+
+		const { flags } = await this.parse(ListLogsCommand);
+
+		const ignoreCache = await flags.ignoreCache;
+		const json = await flags.json;
+		const fileName = await flags.filename;
```

Comment:
Don't need the `await` keywords here.
```suggestion
		const { ignoreCache, json, filename: fileName } = flags;
```